changes/16857.md

```diff
@@ -0,0 +1,14 @@
+# PR 16857: Hotfix / Followup for 16831 (Move plonk_wasm into proof-systems)
+
+## Summary
+
+hotfix / followup to #16820
```

Comment:
As discussed in DM, this CHANGELOG process will also ease the traversal of patches. For instance, here, we could have used
```suggestion
hotfix / followup to [16820](./16820.md)
```
which would have given a direct link to the changelog of the patch !16820.

Nit for this PR, and personal opinion I would try to force myself to apply in the future :).